Runtime_Support/java_support.md

```diff
@@ -1,3 +1,3 @@
 # Java on App Service
 
-For information on our update, support, and runtime retirement policies, please refer to the the official [App Service documentation page for Java](https://docs.microsoft.com/en-us/azure/app-service/containers/configure-language-java#statement-of-support).
+For information on our update, support, and runtime retirement policies, please refer to the the official [App Service documentation page for Java](https://learn.microsoft.com/en-us/azure/app-service/language-support-policy?tabs=linux#java-specific-runtime-statement-of-support).
```

Comment:
please update the link to be: 

https://learn.microsoft.com/azure/app-service/language-support-policy?tabs=linux#java-specific-runtime-statement-of-support

Removing the /en-us/ enables the right thing to happen to present content in the correct locale